README.md

```diff
@@ -125,7 +126,8 @@ __Tolltip__ - If you want to add tooltips on an existing configuration ? Update
         // ...
         "tooltip": [
             { "column": "L7Classification", "label": "L7 Signature" },
-            { "column": "Sum of MB", "format": ",.2s"}
+            { "column": "Sum of MB", "format": ",.2s"},
+            { "column": "duration", "label": "Value", "duration": "h [hrs], m [min], s [sec]"}
```

Comment:
let's call the column something else as it's confusing

Reply:
Actually column name is coming from database, so we can't change the name of the column . If you want then we can change the `duration` property name.

Reply:
correct! But for the sake of the example let's call it something else! I don't want people to think that this column must be named 'duration'